scipp/qens-reduction.ipynb

```diff
@@ -133,16 +329,42 @@
   {
    "cell_type": "markdown",
    "id": "1606596c-407f-4b4a-b95a-a0f70937fbb8",
-   "metadata": {},
+   "metadata": {
+    "editable": true,
+    "slideshow": {
+     "slide_type": ""
+    },
+    "tags": []
+   },
    "source": [
-    "## Transform to energy transfer"
+    "### Transform to energy transfer\n",
+    "\n",
+    "The next step is to compute the measured energy transfer in the sample from time-of-flight and the other coordinates of our data.\n",
+    "We use [scipp.transform_coords](https://scipp.github.io/generated/functions/scipp.transform_coords.html) for this purpose.\n",
+    "\n",
+    "Scippneutron defines some functions for coordinate transformations, see the [documentation](https://scipp.github.io/scippneutron/user-guide/coordinate-transformations.html) for a list.\n",
+    "However, we need to customize the graph for our specific instrument.\n",
+    "We require functions to compute\n",
```

Comment:
Would it help to add an equation, saying e.g. "the energy transfer is defined as .... We have all those but we are missing L2, final wavelength etc..."?

Reply:
Done. Please check if they are readable, in particular for L2.

Reply:
I was thinking more of an equation that describes the energy transfer, which would then make it clearer why we need L2, final wavelength, final energy?
Because right now, apart from final energy, they seem to come a bit out of the blue?

Or instead, show the graph that illustrates that they are needed? But I guess we don't have the graph yet since we need to build it?